inspector/src/components/actionTabs/tabs/propertyGrids/animations/animationCurveEditorComponent.tsx

```diff
@@ -589,6 +610,36 @@ export class AnimationCurveEditorComponent extends React.Component<
     }
   }
 
+  removeKeyframes(points: IKeyframeSvgPoint[]) {
+    if (this.state.selected !== null) {
+      let currentAnimation = this.state.selected;
+
+      const indexesToRemove = points.map((p) => {
+        return {
+          index: parseInt(p.id.split('_')[3]),
+          coordinate: parseInt(p.id.split('_')[2]),
+        };
+      });
+
+      if (currentAnimation.dataType === Animation.ANIMATIONTYPE_FLOAT) {
+        let keys = currentAnimation.getKeys();
+
+        let filteredKeys = keys.filter((_, i) => {
+          if (indexesToRemove.find((x) => x.index === i)) {

```

Comment:
`return !indexesToRemove.find((x) => x.index === i))` is enough. (or `return !indexesToRemove.some((x) => x.index === i))`?)

Reply:
thanks! I use this :)